doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -114,7 +124,7 @@ method to I<name>. This uses the function X509_VERIFY_PARAM_lookup() to
 find an appropriate set of parameters from I<name>.
 
 X509_STORE_CTX_get_num_untrusted() returns the number of untrusted certificates
-that were used in building the chain following a call to X509_verify_cert().
+that were used in building the chain following a call to L<X509_verify_cert(3)>.
```

Comment:
Perhaps somehow mention that if the first untrusted certificate is used as the target with X509_STORE_CTX_verify, it is not counted?